mitiq/mitiq_qiskit/conversions.py

```diff
@@ -269,42 +251,19 @@ def to_qasm(circuit: cirq.Circuit) -> QASMType:
     return circuit.to_qasm()
 
 
-def to_qiskit(
-    circuit: cirq.Circuit,
-    qregs: Optional[List[qiskit.QuantumRegister]] = None,
-    cregs: Optional[List[qiskit.ClassicalRegister]] = None,
-    add_cregs_if_cannot_transform: bool = True,
```

Comment:
When we discussed this @ Mitiq meeting, I totally forgot these were also arguments to `to_qiskit`. I think it's fine to remove them, since we will always transform registers in noise scaling functions via the `noise_scaling_converter`. But pinging @andreamari for a second opinion on removing these arguments. Sorry for the confusion @Aaron-Robertson, mea culpa.

Reply:
@rmlarose I don't see big problems at the moment. PEC is quite delicate with respect to the names of registers, but if all tests pass, it's fine.

Reply:
Sounds good, it's fine to reintroduce if needed.